applications/search/src/pages/search-page/filter-pills/filter-pills.component.jsx

```diff
@@ -69,9 +69,8 @@ export const FilterPills = ({
   }
 
   const clearButtonClass = cx(
-    'btn',
-    'fdk-button',
-    'fdk-button-black-toggle',
+    'fdk-button-clear',
```

Comment:
hvis de stilene kommer fra den nytt designsystemet, da skulle de to kommittene squashes.

Reply:
squashed